models/wholeBrainSeg_Large_UNEST_segmentation/eval/schema.json

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
Please remove the `eval` folder in the PR.

Thanks.

Reply:
Done